### DISCUSSION TITLE
zh/backend/base/router

### DISCUSSION THREAD

**Discussion by giscus** [mineadmin/MineAdmin#477] [Q&A]
# zh/backend/base/router

企业级后台管理系统

https://doc.mineadmin.com/zh/backend/base/router.html

**Comment by chahualao** [comment]
这搞的Java那一套，太繁琐了。

**Reply by zds-s** [reply]
不繁琐，只是为了方便在 `apifox` `apipost` `postman` 各种工具直接导入接口文档

**Reply by zds-s** [reply]
而且不是强制要求，不喜欢 swagger 可以不用。支持各种写法

**Comment by youziyouzishu** [comment]
强制使用swagger注解路由吗

可以使用 RequestMapping 这种注解吗

**Comment by zds-s** [comment]
可以



---Original---
From: ***@***.***&gt;
Date: Wed, Oct 15, 2025 21:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [mineadmin/MineAdmin] zh/backend/base/router (Discussion #477)




 
强制使用swagger注解路由吗
 
可以使用 RequestMapping 这种注解吗
